components/agenda/src/Agenda.svelte

```diff
@@ -329,9 +332,23 @@
     }
   })();
 
-  $: allDayEvents = calendarEvents
-    ?.filter((event): event is DateEvent => "date" in event.when)
-    ?.sort((a, b) => (a.title || "").localeCompare(b.title || ""));
+  let StoredAllDayEvents: DateEvent[] = [];
+  $: Promise.all(Object.values($EventStore)).then((days: DateEvent[][]) => {
```

Comment:
Does this not need to be awaited?

Reply:
I don't think so; because the `$EventStore` updates on new fetches (keys) the reactive binding fires; and because its values are promises, `Promise.all()`ing over the `Object.values` of it will return fulfilled events.